examples/basic-sc/README.md

```diff
@@ -180,18 +140,32 @@ registry:
   password: a-very-hard-to-break-password
 ```
 
+Next the developer needs to give Cartographer permission to create the
+items that will be stamped out in the supply chain. That will be done
+with a service account, whose name will be referenced in the workload.
+You can update [`values.yaml`](./values.yaml) with the desired name:
+
+```yaml
+#@data/values
+---
+# a name that won't collide with other service accounts
+service_account_name: cartographer-example-basic-sc-sa
```

Comment:
This statement made me go to `values.yaml` just to discover that the SA key value was already there. At this point, mentioning that the user would only need to go to the file if needs to modify the SA name, would be useful

Reply:
As noted above, the language is meant to show that updating this is optional.